tools/torchci/clickhouse_query_perf.py

```diff
@@ -9,24 +10,45 @@
 import time
 from datetime import datetime, timedelta, timezone
 from functools import cache
-from typing import Optional
+from typing import Any, Dict, List, Optional
 
+from dotenv import load_dotenv
 from prettytable import PrettyTable
 from torchci.clickhouse import get_clickhouse_client, query_clickhouse
 from torchci.utils import REPO_ROOT
 from tqdm import tqdm  # type: ignore[import]
 
 
+def get_latest_commit() -> str:
+    """Get the SHA of the latest commit."""
+    return subprocess.check_output(["git", "rev-parse", "HEAD"]).decode("utf-8").strip()
+
+
+def get_unstaged_file(file_path: str) -> str:
+    """Get the contents of an unstaged file."""
+    try:
+        with open(file_path, "r", encoding="utf-8") as f:
+            return f.read()
+    except FileNotFoundError:
+        raise FileNotFoundError(f"File not found: {file_path}")
+    except IOError:
+        raise FileNotFoundError(f"Error reading file: {file_path}")
```

Comment:
nit: 
```suggestion
        raise IOError(f"Error reading file: {file_path}")
```

Or even better, just remove the catches since we just re-raise with some non-informative info? (file_path will already be in the error for FileNotFound

Reply:
good catch, removed